docs/docs/advanced/air-gap.md

```diff
@@ -1,142 +1,125 @@
 
-    ```
-    TRIVY_TEMP_DIR=$(mktemp -d)
-    trivy --cache-dir $TRIVY_TEMP_DIR image --download-db-only
-    tar -cf ./db.tar.gz -C $TRIVY_TEMP_DIR/db metadata.json trivy.db
-    rm -rf $TRIVY_TEMP_DIR
-    ```
+The databases are pulled by Trivy using the [OCI Distribution](https://github.com/opencontainers/distribution-spec) specification, which is based on simple HTTPS protocol.
 
-=== "oras >= v0.13.0"
-    Please follow [oras installation instruction][oras].
+## Running Trivy in air-gapped environment
+In an air-gapped environment, you have to tell Trivy on every scan to not attempt to download the latest database files, otherwise the scan will fail. The following flags are relevant:
 
-    Download `db.tar.gz`:
+- `--skip-db-update` to skip updating the main vulnerability database.
+- `--skip-java-db-update` to skip updating the Java vulnerability database.
+- `--offline-scan` to scan Java applications without issuing API requests.
+- `--skip-check-update` to skip updating the misconfiguration database.
 
-    ```
-    $ oras pull ghcr.io/aquasecurity/trivy-db:2
-    ```
-
-=== "oras < v0.13.0"
-    Please follow [oras installation instruction][oras].
+```shell
+trivy image --skip-db-update --skip-java-db-update --offline-scan --skip-check-update myimage
+```
 
-    Download `db.tar.gz`:
+## Self-Hosting
+You can also host the databases on your own OCI registry, in order to avoid having Trivy reaching out of your network.  
 
-    ```
-    $ oras pull -a ghcr.io/aquasecurity/trivy-db:2
-    ```
+First, make a copy of the databases in a container registry that is accessible to Trivy. The databases are in:
+- `ghcr.io/aquasecurity/trivy-db:2`
+- `ghcr.io/aquasecurity/trivy-java-db:1`
+-  `ghcr.io/aquasecurity/trivy-checks:0`
 
-### Download the Java index database[^1]
-Java users also need to download the Java index database for use in air-gapped environments.
+Then, tell Trivy to use the private images:
 
-!!! note
-    You container image may contain JAR files even though you don't use Java directly.
-    In that case, you also need to download the Java index database.
+```shell
+trivy image \
+    --db-repository myregistry.local/trivy-db \
+    --java-db-repository myregistry.local/trivy-java-db \
+    --offline-scan \
+    --checks-bundle-repository myregistry.local/trivy-checks \
+    myimage
+```
 
-=== "Trivy"
+### Authentication
 
-    ```
-    TRIVY_TEMP_DIR=$(mktemp -d)
-    trivy --cache-dir $TRIVY_TEMP_DIR image --download-java-db-only
-    tar -cf ./javadb.tar.gz -C $TRIVY_TEMP_DIR/java-db metadata.json trivy-java.db
-    rm -rf $TRIVY_TEMP_DIR
-    ```
-=== "oras >= v0.13.0"
-    Please follow [oras installation instruction][oras].
+For Trivy DB, configure it in the [same way as for private images](../advanced/private-registries/index.md).
 
-    Download `javadb.tar.gz`:
+For Java DB, you need to run `docker login YOUR_REGISTRY`. Currently, specifying a username and password is not supported.
```

Comment:
The original document was incorrect. The vulnerability DB and Java DB work the same, and it supports only `docker login` now.

Reply:
@knqyf263 Why does Java DB only support authentication via docker login?  When an artifact is loaded, [this](https://github.com/aquasecurity/trivy/blob/main/pkg/oci/artifact.go#L73) method is called, which loads the image with authentication.

Reply:
It's my bad. I was using an old laptop today and referring to an old commit.

Reply:
so just to confirm, vuln db, java db, checks db all support the same authentication, and that is trivy's "private registry" options. right?

Reply:
https://github.com/aquasecurity/trivy/pull/7160/commits/4695eebf7a876ff18c9a56f07e4ef4c141dc7e10

Reply:
Yes